ecommerce/core/management/commands/sync_hubspot.py

```diff
@@ -190,10 +192,9 @@ def _hubspot_endpoint(self, hubspot_object, api_url, method, body=None, **kwargs
         """
         This function is responsible for all the calls of hubspot.
         """
-        expected_methods = ["GET", "POST", "PUT"]
-        api_url = urljoin(HUBSPOT_API_BASE_URL, f"{api_url.lstrip('/')}/{hubspot_object}")
-        if method not in expected_methods:
-            raise ValueError("Unexpected method {}".format(method))
+        api_url = urljoin(HUBSPOT_API_BASE_URL, f"{api_url}/{hubspot_object}")
```

Comment:
Let's use a trailing slash to all URL from settings or variables so we are confident that URL is not broken.
```
        api_url = urljoin(f"{HUBSPOT_API_BASE_URL}/", f"{api_url}/{hubspot_object}")
```

here and below